packages/wonder-blocks-modal/components/modal-backdrop.js

```diff
@@ -36,6 +37,26 @@ export default class ModalBackdrop extends React.Component<Props> {
         }
     };
 
+    componentDidMount() {
+        // Focus the last button in the modal, on the assumption that it'll be
+        // a sensible default action.
+        //
+        // TODO(mdr): Not sure how robust this is; or whether we'll sometimes
+        //     want the default to be something in the modal content, or a
+        //     different button, or something else.
+        const node: HTMLElement = (ReactDOM.findDOMNode(this): any);
+        if (!node) {
+            return;
+        }
+
+        const buttons = node.querySelectorAll("button");
```

Comment:
This is a good start.  We'll want to grab all `input`, `a`, and `button` together and probably grab the first one inside of content and if that doesn't exist the last one inside the footer.  The footer behavior relies on no-one use `flexDirection: "row-reverse"`.

Reply:
For focusing the button we may want to try to figure out a way to search for the "primary" `Button` in the footer.  `Button` can render though as `<button>` or `<a>` (or `<Link>` which I believe renders as `<a>` under the hood).

Reply:
👍🏻 I've got WB-151 written down, which talks about making this cleverer and more robust. Perhaps by reusing the new "focus first descendant" and "focus last descendant" functionality in #71?